src/app/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+import Button from "./Button";
+
+export default {
+  title: "Component/Button",
+  component: Button,
+};
+
+export const Auswahl = (): JSX.Element => <Button>Filmauswahl</Button>;
+export const Film = (): JSX.Element => <Button>Nächster Film</Button>;
```

Comment:
It looks good, but I was thinking if you should use links `a` instead of buttons `button` if you want to redirect to another page.

Reply:
These buttons will probably trigger a function. :)